types/stripe/index.d.ts

```diff
@@ -902,7 +902,7 @@ declare namespace Stripe {
              * For most Stripe users, the source of every charge is a credit or debit card.
              * This hash is then the card object describing that card.
              */
-            source: cards.ICard | bitcoinReceivers.IBitcoinReceiver;
+            source: cards.ICard | bitcoinReceivers.IBitcoinReceiver | bankAccounts.IBankAccount;
```

Comment:
The docs mention credit or debit cards, but bank accounts (among others) are also valid: https://stripe.com/docs/api/charges/create#create_charge-source